examples/js/exporters/GLTFExporter.js

```diff
@@ -1358,11 +1375,17 @@ THREE.GLTFExporter.prototype = {
 
 				if ( geometry.index !== null ) {
 
-					var cacheKey = geometry.uuid + ':' + groups[ i ].start + ':' + groups[ i ].count;
+					var cacheKey = getUID( geometry.index );
+
+					if ( groups[ i ].start !== undefined || groups[ i ].count !== undefined ) {
+
+						cacheKey += ':' + groups[ i ].start + ':' + groups[ i ].count;
+
+					}
 
 					if ( cachedData.attributes.has( cacheKey ) ) {
 
-						primitive.indices = cachedData.attributes.get( geometry.index );
+						primitive.indices = cachedData.attributes.get( cacheKey );
```

Comment:
I've fixed this directly since the other changes might take longer to discuss...